### PR TITLE
Update README for v2, modified constant name

### PR DESCRIPTION
**Pull request by dsgriffin**
* Remove extra build steps resolved by #28 
* Mention /resources vs absolute path option
* Explicitly state support r.e. Cordova/OS versions.
* Renamed file path constant

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,21 +10,17 @@ Cordova-Plugin-Vuforia is a [Cordova][cordova] plugin that uses [Vuforia][vufori
...
 
 
-## Key Features
-- Image recognition using reliable [Vuforia][vuforia] library
-- Comprehensive setup instructions
-
-
 ## Supported Platforms
-Android, iOS
+Android (minimum Android 4), iOS (minimum iOS 8)
```
Comment (mattrayner) [inline README.md]: Android (Minimum 4.0), iOS (Minimum 8.0)

File: README.md [hunk 2]
```diff
@@ -8,21 +10,17 @@ Cordova-Plugin-Vuforia is a [Cordova][cordova] plugin that uses [Vuforia][vufori
...
 
 
 ## Requirements
 Cordova-Plugin-Vuforia requires the following:
-* [NPM][npm]
-* [Cordova][cordova] (Minimum 5.0 - tested to 6.1.1)
-  * `cordova-ios@3.*` instructions on `cordova-ios-3` branch
-  * `cordova-ios@4.*` (default for Cordova 6.*) instructions are below
+* [npm][npm]
```
Comment (mattrayner) [inline README.md]: Renamed for style reasons? Or is NPM incorrect?

File: README.md [hunk 3]
```diff
@@ -49,17 +47,20 @@ navigator.VuforiaPlugin.startVuforia(
 );
 ```
 
-**NOTE**: You will need to replace `YOUR_VUFORIA_KEY` with a valid license key for the plugin to launch correctly.
-
-For testing you can use the `targets/PluginTest_Targets.pdf` file, it contains all four testing targets.
+**NOTES**:
```
Comment (mattrayner) [inline README.md]: Necessary?

File: README.md (hunk 2 above)
Comment (dsgriffin) [inline README.md]: npm is [lowercase](https://www.npmjs.com/)

**Comment by mattrayner** [comment]
We should remove out of date GIFs previously used for iOS instructions
**NOTE:** we need to keep some for the `cordova-ios-3` branch

File: README.md [hunk 4]
```diff
@@ -212,4 +121,4 @@ Cordova-Plugin-Vuforia is licensed under the [MIT License][info-license].
 [shield-travis]: https://img.shields.io/travis/mattrayner/cordova-plugin-vuforia.svg
 [shield-license]: https://img.shields.io/badge/license-MIT-blue.svg
 [shield-bithound]: https://www.bithound.io/github/mattrayner/cordova-plugin-vuforia/badges/score.svg
-[shield-cordova]: https://img.shields.io/badge/cordova%20support-5.*%20--%206.*-blue.svg
+[shield-cordova]: https://img.shields.io/badge/cordova%20support-%206.*-blue.svg
```
Comment (mattrayner) [inline README.md]: Redundant `%20` in shield

File: README.md [hunk 5]
```diff
@@ -49,17 +49,19 @@ navigator.VuforiaPlugin.startVuforia(
...
+* For testing you can use the `targets/PluginTest_Targets.pdf` file inside the plugin folder; it contains all four testing targets.
 
 #### Using your own data
 We know that eventually you're going to want to use your own data. To do so, follow these extra steps.
 
 ##### `www/`
 First, create a `targets/` folder inside `www/` and place your own `.xml`/`.dat`/`.pdf` files inside.
 
+##### `.js`
+You will need to replace the `PluginTest.xml` argument with `www/targets/CustomData.xml` and the `[ 'logo', 'iceland', 'canterbury-grass', 'brick-lane' ]` argument with the specific information for your application. These are here for initial setup and demo purposes only.
```
Comment (mattrayner) [inline README.md]: Does it have to be CustomData.xml? Not sure if I'm just splitting hairs at this  point 😜

**Pull request merged**